tests/test_suvolcanic.F90

```diff
@@ -0,0 +1,139 @@
+program test_suvolcanic
+   use CATChem, fp => cc_rk
+   use testing_mod, only: assert
+   use precision_mod, only: rae
+   implicit none
+
+   type(ConfigType) :: Config
+   type(ChemStateType) :: ChemState
+   type(MetStateType) :: MetState
+   type(DiagStateType) :: DiagState
+   type(SUVolcanicEmissionsStateType) :: SUVolcanicEmissionsState
+   type(GridStateType) :: GridState
+   type(EmisStateType) :: EmisState
+
+   ! Integers
+   INTEGER:: rc          ! Success or failure
+
+   character(len=:), allocatable :: title
+
+   ! Error handling
+   CHARACTER(LEN=512) :: errMsg
+   CHARACTER(LEN=255) :: thisLoc
+   CHARACTER(LEN=255), PARAMETER :: configFile ='Configs/Default/CATChem_config.yml'
+
+
+   thisLoc = 'test_suvolcanic -> at read CATChem_Config.yml'
+   errMsg = ''
+   rc = CC_SUCCESS
+
+   write(*,*) '   CCCCC      A     TTTTTTT   CCCCC  H'
+   write(*,*) '  C          A A       T     C       H        EEEE   M       M'
+   write(*,*) '  C         AAAAA      T     C       HHHHH   E    E  M M   M M'
+   write(*,*) '  C        A     A     T     C       H   H   E EE    M   M   M'
+   write(*,*) '   CCCCC  A       A    T      CCCCC  H   H    EEEEE  M       M'
+   write(*,*) ''
+   write(*,*) ''
+
+   !----------------------------
+   ! Test 1
+   !----------------------------
+
+   ! Read input file and initialize grid
+   call cc_read_config(Config, GridState, EmisState, ChemState, rc, configFile)
+   if (rc /= CC_success) then
+      errMsg = 'Error reading configuration file: ' // TRIM( configFile )
+      call cc_emit_error(errMsg, rc, thisLoc)
+      stop 1
+   endif
+
+
+   title = 'Volcanic Test 1 | Read Config'
+   SUVolcanicEmissionsState%Activate = .false.
+   call print_info(Config, SUVolcanicEmissionsState, MetState, title)
+   write (*,*) '-- '
+   write (*,*) 'Completed ', title
+   write (*,*) '--'
+
+   !----------------------------
+   ! Test 2
+   !----------------------------
+   ! Set number of Volcanic species
+
+   ChemState%nSpeciesSUVolcanic = 2
+   SUVolcanicEmissionsState%Activate = .true.
+
+   ! Meteorological State
+   allocate(MetState%BXHEIGHT(MetState%NLEVS))
+   allocate(MetState%DELP(MetState%NLEVS))
+   MetState%NLEVS = 2
```

Comment:
Looks like this would get the run succeeding
```suggestion
   MetState%NLEVS = 2
   allocate(MetState%BXHEIGHT(MetState%NLEVS))
   allocate(MetState%DELP(MetState%NLEVS))
```